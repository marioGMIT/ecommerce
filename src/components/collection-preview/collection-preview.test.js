import React from 'react';
import { shallow } from 'enzyme';

import CollectionPreview from './collection-preview.component'

describe('CollectionPreview component', () => {
  let wrapper;

  beforeEach(() => {

    const mockProps = {
        
        id: 1,
        title: 'Hats',
        routeName: 'hats',
        items: [
            {
            id: 1,
            name: 'Brown Brim',
            imageUrl: 'https://i.ibb.co/ZYW3VTp/brown-brim.png',
            price: 25
            },
            {
            id: 2,
            name: 'Blue Beanie',
            imageUrl: 'https://i.ibb.co/ypkgK0X/blue-beanie.png',
            price: 18
            }          
            
        ]     
    };

    wrapper = shallow(<CollectionPreview {...mockProps} />);
  });

  it('should render CheckoutItem component', () => {
    expect(wrapper).toMatchSnapshot();
  });

});