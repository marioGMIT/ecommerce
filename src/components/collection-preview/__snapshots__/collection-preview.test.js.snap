// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionPreview component should render CheckoutItem component 1`] = `ShallowWrapper {}`;
